functions.cmake

```diff
@@ -216,6 +216,11 @@ function(_add_executable)
       disable_pax_mprotect(${ARGV})
     else()
       add_executable(${ARGV})
+      list(GET ARGV 0 CLR_CMAKE_CURRENT_TARGET)
```

Comment:
This extraction is not needed, ARGV0 variable can be used to access  that argument. See the usage in the code below.

Reply:
Will fix. Thanks